scripts/package/postremove.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+# Copyright  observIQ, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+remove() {
+  rm -f /usr/lib/systemd/system/observiq-otel-collector.service || {
+    printf 'failed to remove /usr/lib/systemd/system/observiq-otel-collector.service'
+    return
+  }
+  rm -f /etc/init.d/observiq-otel-collector || {
+    printf 'failed to remove /etc/init.d/observiq-otel-collector'
+    return
+  }
+  rm -f /etc/sysconfig/observiq-otel-collector || {
+    printf 'failed to remove /etc/sysconfig/observiq-otel-collector'
+    return
+  }
+  # remove the entire folder
+  # pkg manager will remove most files but this will delete the remaining
+  rm -rf /opt/observiq-otel-collector || {
+    printf 'failed to remove /opt/observiq-otel-collector'
+    return
```

Comment:
On RHEL, this is not working as expected. The package install and removal works great, but I am left with the following:

```
➜  /opt sudo tree observiq-otel-collector
observiq-otel-collector
└── log
    └── collector.log
```

After further inspection, I see this in the output from my package manager.

```
warning: file /etc/init.d/observiq-otel-collector: remove failed: No such file or directory
```

I suspect postremove script failures are considered warnings, because the package manager exits 0 without further error.

I would suggest removing `return` from the `or` operators. If the script fails to delete a file, it should probably continue in this case.

I contemplated suggesting removing the print statements but I think they could be useful.